pages.ja/windows/attrib.md

```diff
@@ -0,0 +1,28 @@
+# attrib
+
+> ファイルまたはディレクトリの属性を表示または変更します
+> 詳しくはこちら:  <https://docs.microsoft.com/windows-server/administration/windows-commands/attrib>.
+
+- 現在のディレクトリ内のファイルの属性を表示します:
+
+`attrib`
+
+- 現在のディレクトリとサブディレクトリにあるファイルの属性を表示します:
+
+`attrib /S`
+
+- 現在のディレクトリとサブディレクトリ内のファイルとディレクトリの属性を表示します:
+
+`attrib /S /D`
+
+- ファイルに読み取り専用属性を追加します:
+
+`attrib +R {{資料.txt}}`
```

Comment:
Are you sure about the translation for `document.txt`, you can use `filename.txt` too:
![Screenshot_20201010_093841](https://user-images.githubusercontent.com/1536740/95655268-6ba52500-0adc-11eb-85b5-13224f55b323.png)

Reply:
Yes, "filename" would make more sense, I have updated it accordingly.